src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Robot;
+
+public class Climber {
+    final int DPAD_UP = 0;
+    final int DPAD_RIGHT = 90;
+    final int DPAD_DOWN = 180;
+    final int DPAD_LEFT = 270;
+    final double DEADBAND = 0.1; // TODO: set
+    final double MAX_SETPOINT = 100; // TODO: set
+
+    Robot robot = null;
+    CANSparkMax rightMotor = new CANSparkMax(1, MotorType.kBrushless);
+    CANSparkMax leftMotor = new CANSparkMax(2, MotorType.kBrushless);
+
+    PIDController rightMotorPID = new PIDController(1.5, 0, 0); // TODO: tune
+    PIDController leftMotorPID = new PIDController(1.5, 0, 0); // TODO: tune
+
+    double rightMotorSetpoint = 0.0;
+    double leftMotorSetpoint = 0.0;
+
+    double leftTriggerAxis;
+    double rightTriggerAxis;
+
+    int dPad;
+
+    boolean isExtended = false;
+
+    String stateString = null;
+
+    public Climber(Robot robot) {
+        this.robot = robot;
+    }
+
+    public void periodic() {
+        dPad = this.robot.controller.getPOV();
+        leftTriggerAxis = this.robot.controller.getLeftTriggerAxis();
+        rightTriggerAxis = this.robot.controller.getRightTriggerAxis();
+
+        if (dPad == DPAD_UP) {
+            rightMotorSetpoint = MAX_SETPOINT;
+            leftMotorSetpoint = MAX_SETPOINT;
+            isExtended = true;
+            stateString = "extended";
```

Comment:
These should be human-readable, not machine-readable (use your words, start with a capital letter)